src/frontend/ui/liste/ExportList.tsx

```diff
@@ -0,0 +1,103 @@
+
+
+import * as XLSX from "xlsx";
+
+import { Paths } from "../../configuration/Paths";
+import { useDependencies } from "../commun/contexts/useDependencies";
+import { RechercheViewModel } from "../home/RechercheViewModel";
+
+export function getCurrentDate() {
+    const today = new Date();
+    const year = today.getFullYear();
+    const month = String(today.getMonth() + 1).padStart(2, "0");
+    const day = String(today.getDate()).padStart(2, "0");
+    const currentDate = `${year}${month}${day}`;
+    return currentDate;
+}
+
+function getType(type: string | undefined) {
+    if (type === "Médico-social") return "Social et Médico-Social"
+    else return type;
+}
+
+async function getComparaisonData(listId: number, order = "", orderBy = "", paths: Paths) {
```

Comment:
Retour Sonar: « Default parameters should be last ».
```suggestion
async function getComparaisonData(listId: number, paths: Paths, order = "", orderBy = "", ) {
```